pages/CreatingAnExploration.html

```diff
@@ -31,23 +30,14 @@
       the essence of your exploration.
     </li>
     <li>
-      <strong>A goal.</strong> What is the exploration supposed to help the
-      player learn?
-    </li>
-    <li>
-      <strong>A category.</strong> You can select one of the predefined
-      categories, or define your own.
-    </li>
-    <li>
-      <strong>A language.</strong> This is the language that your exploration
-      is written in. (In the special case of explorations that teach languages,
-      this should be the language that the exploration is written in, not the
-      language that the exploration is trying to teach.)
+      <strong>An introduction.</strong> What is the exploration supposed to help the
+      learner learn?
     </li>
   </ul>
 
   <p>
-    Once you've done that, click the green button in the bottom right.
-    Congratulations, you've created a new exploration!
+    Once you've done that, a blue button will appear in the bottom left. 
+    Click it to add interactions such as equations, math expressions, maps,
+    buttons and menus. Congratulations, you've created a new exploration!
```

Comment:
Perhaps remove last sentence (since they haven't really finished creating it yet) and link them to a page that has more tips on how to actually create an exploration? At this point you could point to a list of various resources.